common/views/components/MoreInfoLink/MoreInfoLink.js

```diff
@@ -2,29 +2,38 @@
 
 import {spacing, font} from '../../../utils/classnames';
 import Icon from '../Icon/Icon';
+import trackOutboundLink from '../../../utils/track-outbound-link';
 
 type Props = {|
   url: string,
   name: string,
   screenReaderText?: string
 |}
 
-const MoreInfoLink = ({url, name, screenReaderText}: Props) => (
-  <a className={[
-    'flex-inline',
-    'flex-v-center',
-    'plain-link',
-    'font-green',
-    'font-hover-turquoise',
-    font({s: 'HNM4'})].join(' ')} href={url} data-component="MoreInfoLink">
-    <span className='width-1-em'>
-      <Icon name='arrow' extraClasses='icon--green' />
-    </span>
-    <span className={spacing({s: 1}, {margin: ['left']})}>
-      {name}
-      {screenReaderText && <span className='visually-hidden'> {screenReaderText}</span>}
-    </span>
-  </a>
-);
+const MoreInfoLink = ({url, name, screenReaderText}: Props) => {
+  function handleClick(event) {
+    trackOutboundLink(event.currentTarget.href);
+  }
+
+  return (
+    <a
+      onClick={handleClick}
+      className={[
+        'flex-inline',
+        'flex-v-center',
+        'plain-link',
+        'font-green',
+        'font-hover-turquoise',
+        font({s: 'HNM4'})].join(' ')} href={url} data-component="MoreInfoLink">
```

Comment:
Do we want to come up with a few conventions to use in `className`ing?

Reply:
What sort of thing were you thinking? Like alphebetising/ordering of helper functions and strings, or more like 'we use these `x` classes together in several places; should they become a component/class of their own'?

Reply:
I wasn't going that far - I was just thinking about whether we array them, or newline them, how we mix function and string classes.

Maybe a silly conversation that might be best talking about CSS in general.

Reply:
Ah OK. I guess if this could be configured/automated with linter/prettier, that would be nicest from my point of view. I love a convention.